dev/Tiles/metatile.h

```diff
@@ -0,0 +1,60 @@
+#include <stdbool.h>
+#include <stdlib.h>
+#include "../../lib/SMSlib.h"
```

Comment:
I would recommend putting `SMSlib.h` on the include path so you don't need to use relative paths like this.
On my toolchain image I have a single copy of SMSlib in a known location and all my Makefiles have that on the include path so it's just a matter of having an `#include "SMSlib.h"` and you're away. Having a single copy also ensures you don't have lots of slightly different versions of the lib floating around. However, you can also put a copy of both the include file and lib in your project root folder and just add "." as an include path to similar effect.

Reply:
Noted ✅️